docs/REST-API/fliplet-notifications.md

```diff
@@ -147,6 +147,54 @@ You can also target many people at once using any [Sift.js](https://github.com/F
 }
 ```
 
+---
+### Create batch notifications
+
+#### `POST v1/apps/:id/notifications/batch`
+
+Parameters:
+- **notifications** (array of json object)
+    - Parameters for each object inside **notifications** array
+        - Required parameters:
+            - **data** (json object)
+        - Optional parameters:
+            - **scope** (array of json objects or single json object)
+            - **status** (string, defaults to `draft`. Use `published` to make the notification visible to live apps)
+            - **orderAt** (number, defaults to the current time)
+            - **pushNotification** (json object containing payload, delayUntilTimestamp)
+
+Sample request body:
+
+```json
+{
+"notifications":[{
+  "data": {
+    "title": "New article",
+    "message": "John posted an article."
+  },
+  "scope": { "Email": "nick@example.org" },
+  "pushNotification": {
+    "payload": {
+      "title": "New article",
+      "body": "John has posted a new article on the news page. Go check it out!"
+    }
+  }
+},{
+  "data": {
+    "title": "Next article",
+    "message": "John posted new article."
+  },
+  "scope": { "Email": "nick@example.org" },
```

Comment:
You're demonstrating sending an array, so please use a different email? e.g. "jane@example.org"